CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*       @guardian/android-developers
+*       @guardian/devx-operations
```

Comment:
@jorgeazevedo Is this the correct team to add as codeowner here?

Reply:
Close but not quite! I've left a suggestion